app/src/main/java/app/revanced/integrations/settingsmenu/ReVancedSettingsFragment.java

```diff
@@ -126,6 +162,26 @@ public void onCreate(Bundle bundle) {
             sharedPreferences.registerOnSharedPreferenceChangeListener(this.listener);
             this.Registered = true;
 
+            this.screens = new ArrayList<>();
+            this.screens.add((PreferenceScreen) getPreferenceScreen().findPreference("revanced_pref_default_video_quality"));
+
+            final ListPreference wifiQualityList = (ListPreference) screens.get(0).findPreference("revanced_pref_video_quality_wifi");
+            setListPreferenceData(wifiQualityList, true);
+
+            wifiQualityList.setOnPreferenceClickListener(preference -> {
+                setListPreferenceData(wifiQualityList, true);
+                return false;
+            });
+
+            final ListPreference mobileQualityList = (ListPreference) screens.get(0).findPreference("revanced_pref_video_quality_mobile");
+            setListPreferenceData(mobileQualityList, false);
+
+            mobileQualityList.setOnPreferenceClickListener(preference -> {
+                setListPreferenceData(mobileQualityList, false);
+                return false;
+            });
+
+            sharedPreferences.edit().putBoolean("revanced_initialized", true);
```

Comment:
This will not be required, if `ListPreference` contains said property, from the patch PR.